network-api/networkapi/buyersguide/templates/buyersguide/product_page.html

```diff
@@ -328,12 +328,7 @@ <h3 class="h2-heading mb-4">{% trans "Related products" %}</h3>
 {% endblock %}
 
 {% block extra_scripts %}
-  {% if coralTalkServerUrl %}
-    <script src="{{ coralTalkServerUrl }}static/embed.js" async onload="
-      Coral.Talk.render(document.getElementById('coral-talk-stream'), {
-        talk: '{{ coralTalkServerUrl }}'{% if lang_code == "en" %},
-        asset_url: '{{ product.get_parent.url }}products/{{ product.slug }}/'{% endif %}
-      });
-    "></script>
+  {% if use_commento %}
+    <script async defer src="https://cdn.commento.io/js/commento.js"></script>
```

Comment:
Do we need both `async defer`?

Reply:
yes, they do different things: `async` regulates the script load (making it a non-blocking requiest), whereas `defer` regulates loading behaviour (running the code only after the DOM has been constructed, but before DOMContentReady gets signalled)

Reply:
So this essentially means the script file gets loaded asynchronously but the code doesn't run until `DOMContentReady`?

Reply:
until just before `DOMContentReady`, yes